OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/subscriptions/SubscriptionModelStore.kt

```diff
@@ -24,6 +24,8 @@ open class SubscriptionModelStore(prefs: IPreferencesService) : SimpleModelStore
                         model.deviceOS = existingPushModel.deviceOS
                         model.carrier = existingPushModel.carrier
                         model.appVersion = existingPushModel.appVersion
+                        // Don't hydrate device-level subscription status
```

Comment:
This comment seem redundant with `// When hydrating an existing push subscription model, use existing device properties` above.

Reply:
That's very good point, I agree